packages/orchestrator-ui-components/src/rtk/endpoints/startComboBoxOptions.ts

```diff
@@ -0,0 +1,96 @@
+import {
+    ProductDefinition,
+    StartButtonOptionsResult,
+    StartComboBoxOption,
+    WorkflowDefinition,
+    WorkflowTarget,
+} from '@/types';
+
+import { orchestratorApi } from '../api';
+
+const workflowOptionsQuery = `
+    query StartOptions {
+        workflows(first: 1000000, after: 0, filterBy: [{ field: "target", value: "${WorkflowTarget.CREATE}"}]) {
+            page {
+                name
+                description
+                products {
+                    productId
+                    name
+                }
+            }
+        }
+    }
+`;
+
+const taskOptionsQuery = `
+    query StartOptions {
+        workflows(first: 1000000, after: 0, filterBy: [{ field: "target", value: "${WorkflowTarget.SYSTEM}"}]) {
+            page {
+                name
+                description
+            }
+        }
+    }
+`;
+
+export type StartButtonOptionsResponse = {
+    startComboBoxOptions: StartComboBoxOption[];
+};
+
+type WorkflowOptionsResult = StartButtonOptionsResult<{
+    name: WorkflowDefinition['name'];
+    description: WorkflowDefinition['description'];
+    products: Pick<ProductDefinition, 'productId' | 'name'>[];
+}>;
+
+type TaskOptionsResult = StartButtonOptionsResult<
+    Pick<WorkflowDefinition, 'name' | 'description'>
+>;
+
+const startButtonOptionsApi = orchestratorApi.injectEndpoints({
```

Comment:
There is no such API that provides us with startButtonOptions. I think it would be best to stick as close as possible to the data the actual api returns.